test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java

```diff
@@ -1174,7 +1174,10 @@ private synchronized void reset(boolean wipeData) throws IOException {
         for (int i = 0; i < numSharedDedicatedMasterNodes + numSharedDataNodes + numSharedCoordOnlyNodes; i++) {
             Settings nodeSettings = updatedSettings.get(i);
             if (i == autoBootstrapMasterNodeIndex) {
-                nodeSettings = Settings.builder().putList(INITIAL_MASTER_NODES_SETTING.getKey(), masterNodeNames).put(nodeSettings).build();
```

Comment:
Change _masterNodeNames_ verbiage?

Reply:
I also missed this in PR #2150 😅 . I updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2150/commits/717bc112698c9c6efa0b78cf42a3f97a84c567d5 in that PR.